creator-node/src/config.js

```diff
@@ -796,6 +802,18 @@ if (fs.existsSync(pathTo('contract-config.json'))) {
   })
 }
 
+// Set reconfigSPIdBlacklist based off of reconfigSPIdBlacklistString
+config.set(
+  'reconfigSPIdBlacklist',
+  config.get('reconfigSPIdBlacklistString') === ''
```

Comment:
would this be safer as any falsy check e.g. `_.isEmpty()`? would catch null/undefined as well